packages/components/src/components/radio-group/radio-group.ts

```diff
@@ -290,32 +403,83 @@ export default class SdRadioGroup extends SolidElement implements SolidFormContr
                 ?required=${this.required}
                 tabindex="-1"
                 hidden
+                @invalid=${this.handleInvalid}
               />
             </label>
           </div>
-
-          ${this.hasButtonGroup
-        ? html`
-                <sd-button-group part="button-group" exportparts="base:button-group__base">
-                  ${defaultSlot}
-                </sd-button-group>
-              `
-        : defaultSlot}
+          ${defaultSlot}
         </div>
 
-        <slot
-          name="help-text"
-          part="form-control-help-text"
-          id="help-text"
-          class="form-control__help-text"
-          aria-hidden=${hasHelpText ? 'false' : 'true'}
+        <div
+          part="form-control-error-text"
+          id="error-text"
+          class=${cx(
+            'form-control__error-text mt-2 hidden text-error leading-normal',
+            {
+              /* sizes, fonts */
+              sm: 'text-sm',
+              lg: 'text-base'
+            }[this.size]
+          )}
+          aria-hidden=${hasErrorText ? 'false' : 'true'}
         >
-          ${this.helpText}
-        </slot>
+          <slot name="error-text">${this.errorText}</slot>
+        </div>
       </fieldset>
     `;
-    /* eslint-enable lit-a11y/click-events-have-key-events */
   }
+
+  /**
+   * Inherits Tailwind classes and includes additional styling.
+   */
+  static styles = [
+    componentStyles,
+    SolidElement.styles,
+    css`
+      :host {
+        display: block;
+      }
+
+      .form-control-input--vertical ::slotted(sd-radio) {
+        margin-bottom: 8px;
+        display: flex;
+      }
+
+      .form-control-input--vertical ::slotted(sd-radio:last-of-type) {
+        margin-bottom: 0;
+      }
+
+      .form-control-input--horizontal ::slotted(sd-radio) {
+        margin-right: 24px;
+      }
+
+      .form-control-input--horizontal ::slotted(sd-radio:last-of-type) {
+        margin-right: 0;
+      }
+
+      /* Label */
+      .form-control--has-label .form-control__label {
+        display: flex;
+      }
+
+      :host([required]) .form-control--has-label .form-control__label::after {
+        content: '*';
+        margin-left: 2px;
+      }
+
+      /* error text */
+      .form-control--has-error-text .form-control__error-text {
+        display: flex;
+      }
```

Comment:
Should easily be doable with cx(hasError...) or similar.

Reply:
Had an issue with the hidden tailwind class, which is always "!important" -> didn't know that.
But could solve it with a statement. 

So, CSS class is now removed! 🙂